flink-table/flink-table-api-java/src/main/java/org/apache/flink/table/catalog/CatalogManager.java

```diff
@@ -455,6 +457,93 @@ public void createTable(CatalogBaseTable table, ObjectIdentifier objectIdentifie
 			"CreateTable");
 	}
 
+	/**
+	 * Creates a database in a given fully qualified path.
+	 *
+	 * @param catalogName The catalog where database will be created.S
+	 * @param databaseName Name of database to be created.
+	 * @param database The database to be created.
+	 * @param ignoreIfExists If false exception will be thrown if a database exists in the given path.
+	 */
+	public void createDatabase(String catalogName,
```

Comment:
Database should not be created via catalog manager. We should get the catalog first, and just call catalog APIs for db creation. Otherwise, we are basically copying all catalog APIs to catalog manager